### ISSUE TITLE
The serach engine in the website

### ISSUE THREAD

**Issue by ZacZhangzhuo** [compas-dev/compas_view2#182]
**Describe the bug**
![image](https://github.com/compas-dev/compas_view2/assets/57562250/fc3efa9c-10dc-452c-ae2a-ca1b90b086e4)
Maybe need @Licini  for the fix :)

**Comment by Licini** [comment]
Yep this is a bug from the old sphinx template we were using, we will migrate all doc sites to `sphinx_book_theme`, you can take a look at:
https://compas.dev/compas/latest/reference/index.html
https://compas.dev/compas_ifc/latest/index.html

And https://github.com/compas-dev/compas_ifc/blob/main/docs/conf.py for how it is setup, this will be a work we need to do for all compas_extensions soon

**Comment by Licini** [comment]
Even though we are going to migrate to a new repo, but would still be nice that we fix the docs site at this one for a last stable release. @ZacZhangzhuo , since now you are already very familiar with our docs generation. Could you take a look at the compas_ifc repo as a reference for using sphinx_book theme? The code change needed is actually pretty small.
